frontend/appflowy_flutter/lib/plugins/database_view/board/presentation/board_page.dart

```diff
@@ -282,6 +284,26 @@ class _BoardContentState extends State<BoardContent> {
           context
               .read<BoardBloc>()
               .add(BoardEvent.endEditingRow(groupItem.row.id));
+
+          /// Check if the field contains some text
+          var fieldID = fieldController.fieldInfos[0].id;
```

Comment:
Shouldn't check if the `fieldInfos` is empty before you access an index?

Reply:
Sure, I will add an `isEmpty` check as well.